src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -282,6 +282,10 @@ void executeNextCommandTarget()
 
             switch (commandId)
             {
+                case ID.RoslynCommands.GoToBase:
```

Comment:
Since you are using a Modern command, I am not sure anything needs to be added to the AbstractOleCommandTarget.

Reply:
yes, this is not needed. as a matter of fact I believe @jasonmalinowski had a PR to delete this whole class in 16.2?